angular-workspace/projects/ni/nimble-angular/src/directives/table-column/base/nimble-table-column-base.directive.ts

```diff
@@ -47,5 +50,29 @@ export class NimbleTableColumnBaseDirective<T extends TableColumn> {
         this.renderer.setProperty(this.elementRef.nativeElement, 'columnHidden', toBooleanProperty(value));
     }
 
+    public get sortDirection(): TableColumnSortDirection {
+        return this.elementRef.nativeElement.sortDirection;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('sort-direction') public set sortDirection(value: TableColumnSortDirection) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'sortDirection', value);
+    }
+
+    public get sortIndex(): number | null {
+        return this.elementRef.nativeElement.sortIndex;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('sort-index') public set sortIndex(value: NumberValueOrAttribute | null) {
+        if (value === null) {
+            this.renderer.setProperty(this.elementRef.nativeElement, 'sortIndex', null);
+        } else {
+            this.renderer.setProperty(this.elementRef.nativeElement, 'sortIndex', toNumberProperty(value));
```

Comment:
I think we can add a toNullableNumberProperty to our template-value-helpers for this case